etna/ciim/utils.py

```diff
@@ -205,7 +205,9 @@ def resolve_links(markup):
     def link_from_span(span):
         if link := pq(span).attr("link"):
             if iaid := re.match(r"\$link\((?P<iaid>[C0-9]*)\)", link).group("iaid"):
```

Comment:
Maybe one to look at in a separate PR, but the pattern for recognising the `iaid` here is notably different to patterns used elsewhere. This feels much too verbose.

Reply:
Good spot @ababic . There are separate issues here.
1. The api response requires to match the pattern having link in it. A sperate ticket is raised to resolve these links. [DF-51](https://national-archives.atlassian.net/browse/DF-51) It may require further updates.
2. The Iaid patters are different from the Converter.  Nee to know if that pattern using C0-9 were intentional or otherwise.